.packit.yaml

```
@@ -0,0 +1,38 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-hammer_cli_foreman_remote_execution.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-hammer_cli_foreman_remote_execution.spec
+    - .packit.yaml
```

Nit: yamllint always yells at me when I read these files for inconsistent indenting
```suggestion
  - rubygem-hammer_cli_foreman_remote_execution.spec
  - .packit.yaml
```